### ISSUE TITLE
Hierarchical Taxonomies

### ISSUE THREAD

**Issue by jasonvarga** [statamic/ideas#92]
As explained in statamic/cms#74

**Comment by geekish** [comment]
+1, definitely need this.

**Comment by stuartcusackie** [comment]
I just ran into this problem when trying to set up a hierarchical taxonomy called 'Categories'. I've ended up using collection instead, which allows nesting. Now I'm confused... Will I ever have a need for a Statamic taxonomy?

**Comment by tobinski** [comment]
+1. Hope to to see this soon

**Comment by pathly-dev** [comment]
Why ist this still not implemented? Definitely need this!

**Comment by jackmcdade** [comment]
Because of all the other things people also need that we _have_ been implementing. We’ll get to it as soon as we can!

**Comment by pathly-dev** [comment]
@jackmcdade sorry if that sounded like that, but this wasn't supposed to sound like an attack to the statamic devs. Just wanted to push this issue, since it's already 3 years old 😅

**Comment by sliver37** [comment]
This feature is one of the deciding factors when we are starting new projects, when we ask ourselves which CMS will suit the job, anything that needs multiple layers of taxonomies for even moderately complex search/filtering, we need to look at other options. I'm looking forward to when taxonomies are overhauled a little bit so we can use Statamic for that niche of projects as well. :)

**Comment by marcorieser** [comment]
> This feature is one of the deciding factors when we are starting new projects, when we ask ourselves which CMS will suit the job, anything that needs multiple layers of taxonomies for even moderately complex search/filtering, we need to look at other options. I'm looking forward to when taxonomies are overhauled a little bit so we can use Statamic for that niche of projects as well. :)

Exactly this 😅

**Comment by kobe-f8** [comment]
Would be a really nice feature to have!

**Comment by prikkprikkprikk** [comment]
Just chiming in to say that this is something I, too, would like to see being prioritized.

**Comment by arlindmusliu** [comment]
Right now we are having a discussion with a client that needs hierarchical taxonomies.

**Comment by duncanmcclean** [comment]
Please add a 👍  reaction instead of adding comments - that way the Team can sort the feature requests by the 👍  reaction.

**Comment by ExpDev07** [comment]
What is the status on this? I understand it is a big job, but almost ANY site with a category use-case, will have a need for nested categories.

**Comment by duncanmcclean** [comment]
I don't believe it's been worked on yet - based on the number of 👍  I'm guessing it's somewhere high up on the Roadmap but unsure on the ETA.

**Comment by samburgers** [comment]
Am also among the hordes of people awaiting this feature, would be great to get some sort of ETA for those weighing up starting new projects if possible. thanks!

https://github.com/statamic/cms/discussions?discussions_q=nested

**Comment by espenlg** [comment]
I'd love to see this happen! The sooner the better 🙏

**Comment by P4KM4N** [comment]
+1, this is a must have! Hard to tell client to move from WP without this feature.

**Comment by samburgers** [comment]
Is there a sponsorship program to get features like this implemented? Im already a paying pro user, but i would definitely be interested in sponsoring crucial things like this to get worked on

**Comment by marcorieser** [comment]
Now is the time to join the discussion here: https://github.com/statamic/ideas/issues/839

**Comment by braco** [comment]
This was a blocking feature for me unfortunately.

**Comment by hmw-anthonyc** [comment]
Statamic can't really move far beyond tiny simple small business websites without massive cludge work till this is a thing. We've built big sites on it, but it's a huge time-sinking headache and makes us think twice about using this CMS for anything serious, really. For simple small business websites, we'd probably stick with WordPress. This issue is holding Statamic back, big time.

**Comment by jackmcdade** [comment]
@hmw-anthonyc Thanks for the feedback.

**Comment by quadradan** [comment]
I've been watching the development of Statamic from the sidelines for a little while, thinking about how it might replace Drupal in my tech stack. In most ways, Statamic is far superior to Drupal (speed, content structure, restful interface, content authoring experience etc...) Hierarchal Taxonomies are really the only missing feature that's a road block.
I've experimented using Navigation and various field types to see if I could develop a workaround, but to no avail. I'd love to see this feature added in a future version. (Or I'd love to contribute my time as a dev to help make it happen :-)

**Comment by ryanmitchell** [comment]
@quadradan you can use a structured collection and an entries field for this use case. You lose magic routing but its otherwise pretty much the same.

**Comment by albertvisuals** [comment]
I also need this hierarchal taxonomies for almost all projects. I really hope that this could be added to statamic anytime soon.

**Comment by dangelion** [comment]
I am new to Statamic and find it superior to all the CMSs I have worked with, but there are some unforgivable shortcomings that may cause me to abandon it. Hierarchical taxonomies is one of them. Others are:
https://github.com/statamic/ideas/issues/439
https://github.com/statamic/ideas/issues/223

In 2026, not having these features is very serious in my opinion (especially for a paid product).
